README.md

```diff
@@ -0,0 +1,57 @@
+# Table of Contents
+
+- [**Introduction**](#introduction)
+- [**Getting Started**](#getting-started)
+    - [Docker](#Docker)
+- [Acknowledgements](#acknowledgements)
+    - [Contributors](#contributors)
+
+# **Introduction**
+
+All things mystic.
+Currently, the app contains a service for performing a three cards divination reading as JSON.
```

Comment:
**Suggestion:** It would be better to provide a more detailed description of the project in the Introduction section. This will help readers understand the purpose and functionality of the project. [enhancement]
```suggestion
# **Introduction**

Welcome to Cyber Mysticism, a project dedicated to all things mystic. This application provides a service for performing a three cards divination reading, returning the results in a JSON format. The readings are based on the Tarot, a tool of divination that offers insights into the past, present, and future.
```